runtime/runtime_controller.h

```diff
@@ -621,9 +645,6 @@ class RuntimeController : public PlatformConfigurationClient {
 
   bool FlushRuntimeStateToIsolate();
```

Comment:
Can we add a comment that this should be called only once when the Dart isolate is initialized?

Reply:
I've added a boolean to guarantee that this method is only called once.